pkg/mesos/taskio_test.go

```diff
@@ -0,0 +1,228 @@
+package mesos
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"testing"
+
+	mesos "github.com/mesos/mesos-go/api/v1/lib"
+	"github.com/mesos/mesos-go/api/v1/lib/agent"
+	agentcalls "github.com/mesos/mesos-go/api/v1/lib/agent/calls"
+	"github.com/mesos/mesos-go/api/v1/lib/encoding"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAttach(t *testing.T) {
+	containerID := containerWithParent()
```

Comment:
The `containerWithParent` func is used exactly once. I'd prefer to inline the code here.

Reply:
I've inlined it